src/main/java/duckie/command/DeleteCommand.java

```diff
@@ -0,0 +1,53 @@
+package duckie.command;
+
+import java.util.ArrayList;
+
+import duckie.task.*;
+import duckie.Ui;
+import duckie.Storage;
+import duckie.exception.*;
+import duckie.task.*;
+
+/**
+ * Command to delete a task in the TaskList
+ */
+public class DeleteCommand extends Command {
+    private int ind;
+
+    /**
+     * Instantiate the DeleteCommand
+     * @param ind Index of the Task in TaskList to be deleted
+     */
+    public DeleteCommand(int ind) {
+        this.ind = ind;
+    }
+
+    /**
+     * Delete the required Task
+     * @param tasks TaskList containing all the tasks
+     * @param ui Ui to interact with the users
+     * @param storage Storage to write to File
```

Comment:
Should there be a fullstop behind each param line because "Punctuation behind each parameter description"